src/node/node_state.h

```diff
@@ -1441,13 +1454,26 @@ namespace ccf
       // Accept TLS connections, presenting node certificate signed by network
       // certificate
 
-      auto nw = crypto::make_key_pair(network.identity->priv_key);
-      auto csr = node_sign_kp->create_csr(config.subject_name);
-      auto sans = get_subject_alternative_names();
-      auto endorsed_node_cert = nw->sign_csr(network.identity->cert, csr, sans);
+      if (node_cert.empty())
```

Comment:
I'm surprised that this is here. I thought this function was always called at the right times, i.e. always after the node certificate had been endorsed.

Reply:
This is all in flux, I haven't decided where/how to move the calls to `accept_network_tls_connections`. E.g. the version that's currently checked in erroneously re-generates already endorsed certificates.